build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/miscTasks.ts

```diff
@@ -231,6 +231,13 @@ export class TypeValidationTask extends LeafWithFileStatDoneFileTask {
 	private inputFiles: string[] | undefined;
 	private outputFiles: string[] | undefined;
 
+	/**
+	 * Use hashes instead of modified times in donefile.
```

Comment:
Should this comment be updated to consider being removed once AB#7318 is done and it can be a proper TscDependentTask?

Reply:
Added as a remark in the class description.